app-modules/audit/src/Filament/Resources/AuditResource/Pages/ListAudits.php

```diff
@@ -31,6 +35,15 @@ public function table(Table $table): Table
                     ->sortable(),
             ])
             ->filters([
+                SelectFilter::make('change_agent_user')
+                    ->label('Change Agent (User)')
+                    ->options(fn () => User::query()->pluck('name', 'id')->toArray())
```

Comment:
```suggestion
                    ->options(fn (): array => User::query()->pluck('name', 'id')->all())
```

Very minor, but `all()` is more performant than `toArray()` for collections like this